README.md

```diff
@@ -86,6 +86,18 @@ This will download and apply all updates for your system.
 You can find more information about the update workflow in the end
 users section of the [fwupd website](https://fwupd.org).
 
+## Passim
+
+If the [Passim](https://github.com/hughsie/passim/blob/main/README.md) project is also installed
+and enabled, fwupd will re-publish the downloaded metadata file to be served on `0.0.0.0:27500`
+by default.
+
+Other clients on the same network can make use of this via mDNS/LLMNR to reduce network bandwidth
+to configured remotes.
+
+To disable this functionality either set `P2pPolicy=none` in `/etc/fwupd/daemon.conf`, uninstall
+the passim package or use `systemctl mask passim.service` on the terminal.
```

Comment:
Generally this looks good, but I unfortunately don't think it works.  You can't uninstall the package and have it run.

https://github.com/fwupd/fwupd/issues/7173

Reply:
Ohh, that's just Arch -- on Fedora we split out the library from the daemon:

    passim-libs-0.1.7-1.fc40.x86_64
    passim-0.1.7-1.fc40.x86_64
    passim-devel-0.1.7-1.fc40.x86_64

Reply:
Ahh I see.

Reply:
Ohh it's a downstream split -- upstream just blits files to a destdir -- it's up to the packaging system to split them up into -libs, -devel etc.